# M_POS_Local.msg
# ID 182 Pos_Drone_Local_Pos
# ID 184 Set_Desire_Pos_local

# --- Common --- 
# 发送端 编号 
uint8 Send_ID		
# 接收端 编号
uint8 Receive_ID	 

# --- M_POS_GPS ---
# x float
float32 x	
	
# y float
float32 y

# z float
float32 z


